test/Microsoft.IdentityModel.TestUtils/DataSets.cs

```diff
@@ -213,20 +214,6 @@ public static JsonWebKey JsonWebKey2
                 ""x5t"":""kriMPdmBvx68skT8-mPAB3BseeA""
             }";
 
-        public static string JsonWebKeyMixedCaseString =>
```

Comment:
are we losing test coverage by removing this?

Reply:
@keegan-caruso  i folded that with this change: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/2e4f4993b547c99d201577d65fbedd896981329a/test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectSerializationTests.cs#L24